vhdbuilder/scripts/automate_helpers.sh

```diff
@@ -49,7 +49,7 @@ create_pull_request() {
     curl \
         -X POST \
         https://api.github.com/repos/Azure/AgentBaker/pulls \
-        -d '{"head" : "'$3'", "base" : "master", "title" : "Automated PR for '$4'"}' \
+        -d '{"head" : "'$3'", "base" : "master", "title" : "chore: automated PR for '$4'"}' \
```

Comment:
while we're at it could we also include a short description as well?

mainly since we've been trying to steer away from PRs with empty descriptions

Reply:
added description!